src/widgets/_windows.scss

```
@@ -90,6 +90,32 @@ window:not(.popup):not(.menu) {
         }
     }
 
+    &.unified {
```

Probably good to comment this since I wouldn't know where these classes were coming from otherwise

```suggestion
    // Hdy.Window
    &.unified {
```